package.json

```diff
@@ -29,35 +29,42 @@
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-react": "^7.12.1",
     "@babel/preset-typescript": "^7.12.1",
+    "@types/axios": "^0.14.0",
     "@types/cors": "^2.8.8",
     "@types/express": "^4.17.8",
     "@types/jest": "^26.0.14",
+    "@types/memcached": "^2.2.6",
     "@types/pg": "^7.14.5",
     "@types/react": "^16.9.53",
     "@types/react-dom": "^16.9.8",
-    "@types/sinon": "^9.0.8",
+    "@types/supertest": "^2.0.10",
     "@types/winston": "^2.4.4",
+    "axios": "^0.21.0",
     "babel-loader": "^8.1.0",
-    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
     "express": "^4.17.1",
     "husky": "^4.3.0",
     "jest": "^26.6.0",
     "jest-watch-master": "^1.0.0",
     "jest-watch-select-projects": "^2.0.0",
     "jest-watch-suspend": "^1.1.2",
     "jest-watch-typeahead": "^0.6.1",
+    "memcached": "^2.2.2",
     "node-pg-migrate": "^5.8.1",
     "pg": "^8.4.1",
     "pgmock2": "^2.0.5",
     "prettier": "^2.1.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "sinon": "^9.2.0",
     "ts-jest": "^26.4.1",
     "typescript": "^4.0.3",
     "webpack": "^4.44.1",
     "webpack-cli": "^4.1.0",
     "winston": "^3.3.3"
   },
-  "devDependencies": {}
+  "devDependencies": {
```

Comment:
**Comment**: We should move these into `dependencies`, as our app currently does not differentiate the two.

Reply:
Removed Sinon dependency in general. Moved Supertest dev dependency to dependencies.